src/component/page/bruker-detaljer/deltaker-detaljer/DeltakelseInfo.tsx

```diff
@@ -21,17 +20,13 @@ import { Endringsmeldinger } from './endringsmelding/Endringsmeldinger'
 import { FjernDeltakerModal } from './fjern-deltaker-modal/FjernDeltakerModal'
 
 interface DeltakelseInfoProps {
-	deltaker: TiltakDeltakerDetaljer
-	status: DeltakerStatus
-	fjernesDato: Nullable<Date>
+	deltaker: Deltaker
 }
 
 export const DeltakelseInfo = ({
-	deltaker,
-	status,
-	fjernesDato
+	deltaker
 }: DeltakelseInfoProps): React.ReactElement => {
-	const [ reloadEndringsmeldinger, setReloadEndringsmeldinger ] = useState(true)
+	const [ reloadEndringsmeldinger, setReloadEndringsmeldinger ] = useState(false)
```

Comment:
Denne er endret til false for at vi skal bruke endringsmeldingene som følger med deltakeren som utgangspunkt. Hvis man trigger en ny endringsmelding ved f.eks. å forlenge deltakelsen så hentes endringsmeldingene på nytt fra det gamle endepunktet for å hente endringsmeldinger i amt-tiltak.